### PR TITLE
Submissions screen

### PR DESCRIPTION
**Pull request by k1dp-pge**
Adds switch between assignment and submissions screens. Right now assignments screen is blank and submissions all have the same dummy data. No buttons/table cells are functional.

### CODE REVIEW

File: CAP/ViewControllers/PGESubmissionsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        return 5
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return rowHeight
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "submissionCell", for: indexPath) as! PGESubmissionTableViewCell
+        cell.titleLabel.font = UIFont.caBodyTitlesFont()
```
Comment (ahh-pge) [inline CAP/ViewControllers/PGESubmissionsViewController.swift]: use dependency injection here.  We want to move towards having cells handle their own formatting.

File: CAP/ViewControllers/PGESubmissionsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            if section == 0 {
+                return getPlistValueFor(key: "SubmissionsDraftTitle")
+            }
+            return getPlistValueFor(key: "SubmissionsRecentTitle")
+        }
+        headerView.headerLabel.text = headerTitle
+        headerView.headerLabel.font = UIFont.caSectionDividerFont()
+        headerView.headerLabel.textColor = UIColor.caPGEBlueMed
+        headerView.underline.backgroundColor = UIColor(red: 7/255, green: 103/255, blue: 154/255, alpha: 1.0)
+        headerView.sortIcon.isHidden = section == 0
```
Comment (ahh-pge) [inline CAP/ViewControllers/PGESubmissionsViewController.swift]: Here as well.  Have the PGESubmissionHeaderView set fonts and colors.

**Comment by Gar23** [comment]
Can you change VC names with this PR, when you include Drew' changes

**Comment by k1dp-pge** [comment]
@Gar23 Working on it now (:

File: CAP/ViewControllers/PGESubmissionsViewController.swift (hunk 1 above)
Comment (k1dp-pge) [inline CAP/ViewControllers/PGESubmissionsViewController.swift]: I think I did what you're asking. Let me know if you were expecting something different.

**Pull request merged**